doh-server/doh-server.conf

```diff
@@ -4,6 +4,10 @@ listen = [
     "[::1]:8053",
 ]
 
+# HTTP local address for upstream DNS
```

Comment:
I guess we don't need to put the word "HTTP" here?

Reply:
You're right, I'll update that shortly.

What do you think about me question on error-handling above?